contribute.json

```diff
@@ -19,7 +21,8 @@
     },
     "urls": {
         "prod": "https://support.mozilla.org/",
-        "stage": "https://support.allizom.org/"
+        "stage": "https://support.allizom.org/",
+        "dev": "https://support-dev.allizom.org/"
```

Comment:
Let's remove dev. I would treat that instance as ephemeral. Even the database is not similar to stage.